packages/mongo/tests/hooks/update_local.js

```
@@ -0,0 +1,241 @@
+    })
+  }
+)
+
+Tinytest.addAsync(
+  'update - a previous document should be present if fetchPrevious is true',
+  async function (test) {
+    const collection = new Mongo.Collection(null)
+
+    async function start () {
+      collection.after.update(
+        function (userId, doc, fieldNames, modifier) {
```

ESLint

[error] 'userId' is defined but never used.

(no-unused-vars)